theme_kea/models/theme_kea.py

```diff
@@ -6,7 +6,7 @@ class ThemeKea(models.AbstractModel):
 
     def _theme_kea_post_copy(self, mod):
         self.disable_view('website.footer_custom')
-        self.enable_view('website.template_footer_call_to_action')
-
+        self.enable_view('website.template_footer_minimalist')
+        
```

Comment:
Configure your editor correctly and review your code before sending it to me: no spaces at the end of lines please, thanks :+1: